packages/webapp/src/containers/hooks/useCurrencySymbol.js

```diff
@@ -0,0 +1,9 @@
+import { useSelector } from 'react-redux';
+import { currencySelector } from '../userFarmSlice.js';
+import commonCurrency from '../AddFarm/currency/commonCurrency.json';
+
+// TODO: Test code splitting on commonCurrency.json
```

Comment:
The whole json file is imported when the hook only needs to loop up a symbol. 
I don't know how vite json code splitting works. 
This can be a tech debt story